### ISSUE TITLE
Job worksheet N/A values

### ISSUE THREAD

**Issue by VitalijTetervov** [OhmSweetOhm/CobieQcReporter#11]
The following parameters in the JOB worksheet accept N/A values: 

Duration
DurationUnit
Start
TaskStartUnit
Frequency
FrequencyUnit
TaskNumber
Priors
ResourceNames

According to NBIMs v3 constraints, 'n/a' is not acceptable value.

**Comment by StewartGH1970** [comment]
If you look at Annex A page 34 it states all the acceptable values ie Duration = 0, Description = n/a etc

**Comment by VitalijTetervov** [comment]
Agree. It does state acceptable values. The issue here is that QC reporter accepts N/A, where acceptable value is specified as '0':

Example: 
Property | NBIMs | COBieQCRules

Start | Value must be provided or '0' | validStringOrNA
TaskStartUnit | Value must be found in referenced Foreign Key List | validStringOrNA

My question is: why QCReporter accepts N/A? Any help is highly appreciated.

**Comment by OhmSweetOhm** [comment]
Hi,

I will have to dig further to see if there was any reason this was intentional.  If it is a mistake on my part then it can be corrected in the next release.
